value/pom.xml

```diff
@@ -148,9 +148,18 @@
                 </filter>
               </filters>
 	      <relocations>
-		<!-- We don't relocate Apache classes because they often load classes from strings,
-                     and the relocator doesn't know to rewrite those strings. This applies to both
-                     Velocity and Commons. We also don't minimize Velocity for similar reasons. -->
+		<!-- We don't relocate Velocity classes because they often load classes from
+		     strings, and the relocator doesn't know to rewrite those strings.
```

Comment:
I'm seriously tempted to write a template engine from scratch that is compatible with the subset of Velocity that I use. Velocity is great in that it's very widely used and has a great deal of tooling, but it's terribly overengineered for what we're using it for.